packages/angular/ng-package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "node_modules/ng-packagr/ng-package.schema.json",
+  "dest": "dist",
+  "lib": {
+    "entryFile": "src/index.ts",
+    "umdModuleIds": {
+      "@sentry/browser": "Sentry",
+      "@sentry/utils": "utils"
```

Comment:
utils shouldn't be exposed globally in umd, they are free to generate their own from `@sentry/utils`.

Taking the `utils` namespace might be problematic behaviour

Reply:
I'm not entirely sure what to put here to be honest. If I delete the `@sentry/utils` entry, I get the following compiler warning: 

```
WARNING: No name was provided for external module '@sentry/utils' in output.globals – guessing 'utils'
`
``

My understanding is that the `utils` namespace would then still be taken by the compiler's guess. Or should I simply give them a different namespace, say `sentryUtils`?

Reply:
I found this - https://stackoverflow.com/a/61442340

Let's do something like `Sentry.util` then?